src/components/table/_table.scss

```diff
@@ -133,7 +181,6 @@
                   // Cell header label pseudo element
                   &::before {
                     content: attr(data-label);
-                    display: inline-block;
```

Comment:
Same as above.

Reply:
That line `display: inline-block;` doesn't exist here ;-)